### ISSUE TITLE
The attribute structrual

### ISSUE THREAD

**Issue by amelieled** [runtimeverification/michelson-semantics#341]
In the file `michelson-semantics/michelson.md`, someone can read:
```
rule {}                 => .K      [structrual]
```
Do you mean `structural`?

**Comment by sskeirik** [comment]
Yes, that was misspelled. Feel free to file a PR that corrects this (or any other typos) and I will approve it.

**Comment by amelieled** [comment]
Done!
(No any other typos in the K semantic, I think).
